app/src/main/java/cm/aptoide/pt/install/InstallManager.java

```diff
@@ -584,19 +595,33 @@ public boolean wasAppEverInstalled(String packageName) {
   }
 
   public void moveCompletedDownloadFiles(Download download) {
-    for (final FileToDownload fileToDownload : download.getFilesToDownload()) {
+    if (!haveFilesBeenMoved(download)) {
```

Comment:
Why do we have to loop two times? Why don't we verify in the same loop?

And what happens if one file has been moved and others didn't? With this method, it will still try to move that one file that was already moved.